test/test_suite.bats

```diff
@@ -283,7 +283,8 @@
 @test "Ownership and permissions of pihole-FTL.db correct" {
   run bash -c 'ls -l /etc/pihole/pihole-FTL.db'
   printf "%s\n" "${lines[@]}"
-  [[ ${lines[0]} == "-rw-r--r-- 1 pihole pihole"* ]]
+  [[ ${lines[0]} == *"pihole pihole"* || ${lines[0]} == *"pihole   pihole"* ]]
```

Comment:
`x86_64-musl` is built on `busybox` which has a slightly different variant of `ls` displaying three, instead of one, spaces between the user and group names.